sandbox/public/e2e/index.html

```diff
@@ -29,12 +29,40 @@
           : decodeURIComponent(results[1].replace(/\+/g, " "));
       }
 
+      function generateEmail(length, domain) {
+        const chars = [..."abcdefghijklmnopqrstuvwxyz0123456789"];
+        return [...Array(length)].map(i=>chars[Math.random()*chars.length|0]).join`` + '@' + domain;
+      }
+
+      function isSetCustomerIds() {
+        return (getUrlParameter("setCustomerIds") == "true");
+      }
+
       alloy("configure", {
+        thirdPartyCookiesEnabled: isSetCustomerIds() ? true : false,
```

Comment:
I would a comment here to remove this hack once we've figured out the issue ;)

Reply:
@jfkhoury Done.